frontend/src/App.js

```diff
@@ -1,23 +1,17 @@
-import logo from './logo.svg';
 import './App.css';
-
+import { BrowserRouter as Router, Route, Link, Routes } from "react-router-dom";
+import { LogIn } from './components/LogIn/login';
+import { SignUp } from './components/SignUp/signup';
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Router>
+          <Routes>
+              <Route exact path="/" element={<h1>Home Page</h1>} />
+              <Route exact path="/SignUp" element={<SignUp />} />
```

Comment:
For paths we want to use lower kebab case. Please change the path to to "sign-up".